content/applications/marketing/events/event_templates.rst

```diff
@@ -0,0 +1,214 @@
+
+Start by providing a :guilabel:`Name` for this booth in the corresponding field at the top of the
+pop-up window.
+
+Then, select an appropriate :guilabel:`Booth Category` from the drop-down field below. Booth
+categories can be created and modified from the *Booth Categories* page in the *Events* application,
+which is accessible by navigating to :menuselection:`Events app --> Configuration --> Booth
+Categories`.
+
+.. tip::
+   A :guilabel:`Booth Category` can be created directly from this field on the :guilabel:`Create
+   Booths` pop-up window. To accomplish that, type the name of the new booth category in the
+   :guilabel:`Booth Category` field, and select either :guilabel:`Create` or :guilabel:`Create and
+   edit...` from the resulting drop-down menu.
+
+   Clicking :guilabel:`Create` merely creates the category, which can (and should be) customized at
+   a later date. Clicking :guilabel:`Create and edit...` reveals a new :guilabel:`Create Booth
+   Category` pop-up window, from which the category can be configured in a number of different ways.
+
+   .. image:: event_templates/create-booth-category-popup.png
+      :align: center
+      :alt: The Create Booth Category pop-up window in the Odoo Events application.
+
+   From this pop-up window, proceed to name the :guilabel:`Booth Category`, modify its
+   :guilabel:`Booth Details` settings, configure its :guilabel:`Sponsorship` options (if
+   applicable), and leave an optional :guilabel:`Description` to explain any pertinent details
+   related to this specific category of booths.
+
+   There is also the option to add a photo/visual representation of the booth category, via the
+   :guilabel:`camera` icon in the upper-right corner.
+
+   When all desired configurations are complete, click the :guilabel:`Save & Close` button.
+
+   The same configurations and options are available by navigating to :menuselection:`Events app -->
+   Configuration --> Booth Categories`, and clicking :guilabel:`New`.
+
+Once the desired :guilabel:`Booth Category` is selected, the remaining fields on the
+:guilabel:`Create Booths` pop-up window (:guilabel:`Currency`, :guilabel:`Product`, and
+:guilabel:`Price`) autopopulate, based on information configured for that selected :guilabel:`Booth
+Category`.
+
+.. note::
+   These fields **cannot** be modified from the :guilabel:`Create Booths` pop-up window. They can
+   only be modified from the specific booth category form page.
+
+When all desired configurations are complete, click :guilabel:`Save & Close` to save the booth, and
+return to the event template form. Or, click :guilabel:`Save & New` to save the booth, and start
+creating another booth on a fresh :guilabel:`Create Booths` pop-up window. Click :guilabel:`Discard`
+to remove all changes, and return to the event template form.
+
+Once the booth has been saved, it appears in the :guilabel:`Booths` tab on the event template form.
+
+Use event templates
```

Comment:
I like how you included how to use the template after showing how to set it up! (: